haystack/preview/components/websearch/utils.py

```diff
@@ -0,0 +1,39 @@
+from typing import List, Optional, Any
+
+from haystack.preview import Document
+
+
+def calculate_ranking_scores(list_items: List[Any], boost_first_factor: Optional[int] = None) -> List[float]:
+    """
+    Assigns scores to items in a list based on their rank position and ensures that the scores add up to 1.
+    :param list_items: The list of items to score.
+    :param boost_first_factor: The factor to boost the score of the first item by.
```

Comment:
Can we expand this docstring a bit? It's not very clear why one would boost the first result, by how much, what does it entails, etc

Reply:
I removed it, not needed now